tests/providers/google/cloud/hooks/test_dataprep.py

```diff
@@ -29,18 +31,29 @@
 
 class TestGoogleDataprepHook(TestCase):
     def setUp(self):
-        self.hook = dataprep.GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
+        with mock.patch(
+            "airflow.providers.google.cloud.hooks.dataprep.GoogleDataprepHook._token"
+        ) as token:
+            extra = {"token": token}
+
+        with mock.patch(
+            "airflow.providers.google.cloud.hook.dataprep.GoogleDataprepHook.get_connection"
+        ) as conn_mock:
+            conn_mock.return_value = Connection(
+                conn_id="dataprep_conn_id", extra=json.dumps(extra)
+            )
+            self.hook = dataprep.GoogleDataprepHook(dataprep_conn_id="dataprep_conn_id")
 
     @patch("airflow.providers.google.cloud.hooks.dataprep.requests.get")
     @patch("airflow.providers.google.cloud.hooks.dataprep.GoogleDataprepHook._token")
-    def test_mock_should_be_called_once_with_params(self, mock_hook, mock_get_request):
+    def test_mock_should_be_called_once_with_params(self, mock_token, mock_get_request):
```

Comment:
I think there's no need to mock token any more if we mock connection, that's why I would suggest having the global constant `TOKEN`